great-idea-website/index.html

```diff
@@ -15,49 +15,87 @@
 </head>
 
   <body>
-
-  Services
-  Product
-  Vision
-  Features
-  About
-  Contact
-
-  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-  Innovation
-  On
-  Demand
-
-  Get Started
-
-  <img src="img/header-img.png" alt="Image of a code snippet.">
-
-  Features
+    <header>
+    <div class="container">
+      <nav>
+    <a href="#"> Services </a>
+    <a href="#">Product</a>
+    <a href="#"> Vision</a>
+    <a href="#">Features</a>
+    <a href="#">About</a>
+    <a href="#">Contact</a>
+        </nav>
+      <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+    </div>
+</header>
+
+<section class=all-cta>
+  <div class = ctaContent>
+    <h1>
+      Innovation</br>
+      On</br>
+      Demand</br>
+</h1>
+    <button>Get Started</button>
+  </div>
+  <img class = cta-img src="img/header-img.png" alt="Image of a code snippet."/>
+
+</section>
+
+<hr></hr>
```

Comment:
Good job with the `<hr>` tag for the break!  I think it's the first time I've seen that tag